### PR TITLE
Login

### PR DESCRIPTION
**Pull request by alicemcneil**

### CODE REVIEW

File: login/login_confirm.php [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<?php
+session_start();
+if (empty($_SESSION['loggedIn']) || $_SESSION['loggedIn'] != true)
+{
```
Comment (mporam) [inline login/login_confirm.php]: PSR2 says `{` of an if should go on the first line:
`if (condition) {`

File: login/login_form.php [hunk 2]
```diff
@@ -0,0 +1,16 @@
+<link rel="stylesheet" type="text/css" href="login.css">
```
Comment (mporam) [inline login/login_form.php]: no doctype, head or body?

File: login/login_func.php [hunk 3]
```diff
@@ -0,0 +1,20 @@
+<?php
+define('USERNAME', 'alice');
+define('PASSWORD', '$2y$10$onfrLvU36ruTZDvRxUPX5eLDkI.bXZJYsaDfiF.mQL7MZfj6zgTXK'); //pw: 1234
```
Comment (mporam) [inline login/login_func.php]: probably dont publish your password

File: login_form.php [hunk 4]
```diff
@@ -0,0 +1,16 @@
+<link rel="stylesheet" type="text/css" href="login.css">
```
Comment (mporam) [inline login_form.php]: why are there two login forms?!?

**Review by mporam — CHANGES_REQUESTED** [review]

File: login/login.css [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+    text-decoration: none;
+    display: block;
+    font-size: 20px;
+    background-color: #2b7a78; /* Green */
+    border-radius: 10px;
+    margin: 40px auto;
+}
+h3 {
+    font-size: 25px;
+    /*padding-bottom: 10px;*/
```
Comment (neilbulloch) [inline login/login.css]: delete comment

File: login/login_form.php (hunk 2 above)
Comment (alicemcneil) [reply login/login_form.php]: did lots of html structure improvements 👍

**Review by mporam — APPROVED** [review]

**Pull request merged**